network/nodenetwork/utils_test.go

```diff
@@ -15,52 +15,36 @@
  *
  */
 
-package packets
+package nodenetwork
 
 import (
 	"testing"
 
 	"github.com/insolar/insolar/core"
-	"github.com/insolar/insolar/network/transport/packet/types"
-	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/assert"
 )
 
-func checkGetAndSetPacketHeader(t *testing.T, packet PacketRoutable, header *RoutingHeader) {
-	err := packet.SetPacketHeader(header)
-	require.NoError(t, err)
+func Test_diffList(t *testing.T) {
```

Comment:
Please check corner cases as well, e.g. old is empty, new is empty, both are empty.

Reply:
reworked function and tests